pocs/mount/mount.py

```diff
@@ -501,10 +501,10 @@ def get_ms_offset(self, offset, axis='ra'):
         """ Get offset in milliseconds at current speed
 
         Args:
-            offset (float): Offset in arcseconds
+            offset (astropy.units.Angle): Offset in arcseconds
```

Comment:
Support for using type annotations. :-)

Reply:
Actually the astropy units was one of the things I was concerned about. I'm hoping it works seamlessly for them.